dev/run.sh

```
@@ -0,0 +1,246 @@
+# new way to start rsyslog since bookworm
+/usr/sbin/rsyslogd && echo -n " rsyslog started! ✅..." || echo -n " rsyslog failed to start! ❌..."
+
+service cron start
+echo "$prefix  Restarting services : rsyslob, cron done! ✅"
+
+
+echo "$prefix  Launching cron_monkey_patch.sh..."
+test -e /opt/publik/scripts/startup/cron_monkey_patch.sh && ( /opt/publik/scripts/startup/cron_monkey_patch.sh && echo " done! ✅" || echo " failed! ❌" ) || echo " skipped! 🚫"
+
+
+hobo_agent_altered_line=$(grep "hobo_provision" $hobo_agent_file)
```

Shellcheck (0.11.0)

[warning] hobo_agent_file is referenced but not assigned.

(SC2154)